VocaDbModel/Service/Search/AlbumSearch/AlbumSearchSort.cs

```diff
@@ -11,9 +11,7 @@ public class AlbumSearchSort
 	{
 		private static IQueryable<Album> AddReleaseRestriction(IQueryable<Album> criteria)
 		{
-			return criteria.Where(a => a.OriginalRelease.ReleaseDate.Year != null
-				&& a.OriginalRelease.ReleaseDate.Month != null
-				&& a.OriginalRelease.ReleaseDate.Day != null);
+			return criteria.Where(a => a.OriginalRelease.ReleaseDate.Year != null);
 		}

```

Comment:
This method is only used by https://github.com/VocaDB/vocadb/blob/465f446275922de04689739b33ade382bed8a3f6/VocaDbModel/Service/Search/AlbumSearch/AlbumSearchSort.cs#L17
which is only used by https://github.com/VocaDB/vocadb/blob/465f446275922de04689739b33ade382bed8a3f6/VocaDbModel/Service/AlbumService.cs#L52
which is only used by https://github.com/VocaDB/vocadb/blob/465f446275922de04689739b33ade382bed8a3f6/VocaDbModel/Service/AlbumService.cs#L45
which is only used by https://github.com/VocaDB/vocadb/blob/465f446275922de04689739b33ade382bed8a3f6/VocaDbModel/Service/AlbumService.cs#L124
which is not used. Can we remove all of these methods?

Reply:
Resulting in the whole deletion of the `vocadb/VocaDbModel/Service/Search/AlbumSearch/AlbumSearchSort.cs` file, if I'm understanding correctly?

Reply:
Yes.